<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SdNdeCoDeviceWidget</class>
 <widget class="QWidget" name="C_SdNdeCoDeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_SdNdeCoDeviceWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="stw::opensyde_gui::C_SdNdeCoDeviceEdsWidget" name="pc_DevEdsWidget" native="true"/>
   </item>
   <item>
    <widget class="stw::opensyde_gui::C_SdNdeCoDeviceConfigWidget" name="pc_DevConfigWidget" native="true"/>
   </item>
   <item>
    <widget class="stw::opensyde_gui::C_SdNdeCoPdoWidget" name="pc_DevPdoWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw::opensyde_gui::C_SdNdeCoDeviceEdsWidget</class>
   <extends>QWidget</extends>
   <header>C_SdNdeCoDeviceEdsWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui::C_SdNdeCoDeviceConfigWidget</class>
   <extends>QWidget</extends>
   <header>C_SdNdeCoDeviceConfigWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui::C_SdNdeCoPdoWidget</class>
   <extends>QWidget</extends>
   <header>C_SdNdeCoPdoWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
